configuration/controller/mariadb.sh

```diff
@@ -35,16 +34,15 @@ function setup_root_pass {
     PASS=`pass_esc $1`
     do_sql_req "UPDATE mysql.user SET Password=PASSWORD('$PASS') WHERE User='root';"
     do_sql_req "FLUSH PRIVILEGES;"
-    cat << EOF > ~/.my.cnf
-[mysql]
-user=root
-password=$PASS
-[mysqldump]
-user=root
-password=$PASS
-EOF
+    cat <<-EOF > ~/.my.cnf
```

Comment:
it might cause problems for the one who has non-default vim or git  config, like `set expandtab`